lte/gateway/c/core/oai/test/ngap/BUILD.bazel

```diff
@@ -0,0 +1,82 @@
+# Copyright 2021 The Magma Authors.
+
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
```

Comment:
`lte/gateway/c/core/oai/test/ngap/ngap_test.cpp` is not referenced here
* seems to be a test suite for the package
* logging is configured here, though - is this relevant for test output?
* if this file is not used in Bazel, do we need a tech debt task where files are listed for deletion if cmake becomes obsolete?

Same holds for
* `lte/gateway/c/core/oai/test/s1ap_task/s1ap_test.cpp`
* `lte/gateway/c/core/oai/test/sgw_s8_task/sgw_s8_test.cpp`

Reply:
Indeed, top level test "wrappers" are to be dropped - I have a GH issue for this in the Post-Bazelification section of the Bazel MME GH Issue.  It will be simpler to accomplish once CMake is gone (was my thought). 
 https://github.com/magma/magma/issues/11736

Reply:
Oh cool, its already tracked :)